app/views/users/backup_code_setup/index.html.slim

```diff
@@ -0,0 +1,54 @@
+- title t('forms.backup_code.title')
+- btn_cls = 'ml2 btn-border ico ico-print text-decoration-none'
+
+h1.h3.my0.mt0 = t('forms.backup_code.info')
+p.mt-tiny.mb3 == t('forms.backup_code.instructions')
+.mb3
+  fieldset.m0.p0.rounded-xl
+    ul.list-reset
+      li.px-12p.mb1.border-none.mt1.m2
+        .inline-block.h1.bold = t('forms.backup_code.subtitle1')
+        .inline-block = t('forms.backup_code.subinfo1')
+      li.mb0.border-top
+        .alert-warning.p1.bold = \
+          image_tag(asset_url('alert/warning.svg'), size: '20', class: 'mr1') + \
+          t('forms.backup_code.warn')
+        table.ml4
+          tr
+            td
+              br
+              strong
+                - @codes[0..@codes.length / 2 - 1].each do |code|
```

Comment:
Style: we're using the table to format rather than present a proper table. I'd be more comfortable with this being a straight up table. Or, if not a table, then use the grid to create a display that works on large displays as well as small handhelds.

Reply:
I tried the grid but it didn't copy properly when you highlight the codes and copy/paste so I opted to use a table.